packages/frontend/src/components/RefreshButton.tsx

```diff
@@ -3,21 +3,22 @@ import React from 'react';
 import { UseQueryResult } from 'react-query';
 import { ApiError, ApiQueryResults } from 'common';
 import { useExploreConfig } from '../hooks/useExploreConfig';
-import { rudderAnalytics } from './Analytics';
+import { useApp } from '../providers/AppProvider';
 
 type RefreshButtonProps = {
     queryResults: UseQueryResult<ApiQueryResults, ApiError>;
 };
 export const RefreshButton = ({ queryResults }: RefreshButtonProps) => {
     const { validQuery } = useExploreConfig();
     const { refetch, isFetching } = queryResults;
+    const { rudder } = useApp();
     return (
         <Button
             intent="primary"
             style={{ height: '40px', width: 150, marginRight: '10px' }}
             onClick={() => {
                 refetch();
-                rudderAnalytics.track('query_executed');
+                rudder.track('query_executed');
```

Comment:
Should this be tracked in the BE instead?

Reply:
Yes it could! This is a relic from when we only had FE. I'm going to iterate with Katie on another PR